app/controllers/PlutoController.scala

```diff
@@ -31,4 +43,34 @@ class PlutoController(
       Ok(Json.toJson(project))
     }}
   }
+
+  def resendAtomMessage(id: String) = APIHMACAuthAction {
+    val settings = com.gu.media.Settings(this.config)
+    val pluto = new PlutoUploadActions(awsConfig)
+    try {
+      val atomContent = getPreviewAtom(id)
+      val atom = MediaAtom.fromThrift(atomContent)
+      val versionWithId = MediaAtomHelpers.getCurrentAssetVersion(atom) match {
+        case Some(versionNumber)=>id + s"-$versionNumber"
+        case None=>
+          log.error("Requested re-index on an atom with no currentAssetVersion, this could indicate a problem.")
+          id
+      }
+
+      atom.plutoData match {
+        case Some(data)=>
+          pluto.sendToPluto(PlutoResyncMetadataMessage.build(
+            versionWithId,
+            atom,
+            awsConfig,
+            "system@video.gutools.co.uk"
+          ))
+          Ok(Json.toJson(atom))
+        case None=>
+          BadRequest(s"$id does not have pluto data attached")
+      }
+    } catch {
+      case excep:Throwable=>InternalServerError(excep.toString)
```

Comment:
could we follow the pattern of other actions in returning `commandExceptionAsResult`, e.g here - https://github.com/guardian/media-atom-maker/blob/master/app/controllers/Api2.scala#L51-L58